content/getting-started/icerpc-protobuf-tutorial/client-tutorial.md

```diff
@@ -4,7 +4,7 @@ title: Writing your first IceRPC + Protobuf client in C#
 
 This tutorial is the second part of a two part series that shows how to create a
 complete application with IceRPC + Protobuf for C#. We start from scratch—you just need to
-have the .NET 8 SDK installed on your computer.
+have the .NET 8 SDK or .NET 9 SDK installed on your computer.
```

Comment:
I would not repeat SDK:
... the .NET 8 or .NET 9 SDK installed on your computer.

Reply:
I don't like the phrasing, it sounds like .NET 8 OR .NET 9 SDK. What about .NET 8 or 9 SDK.

Reply:
ok